app/mailers/emailer.rb

```diff
@@ -17,27 +17,33 @@ def submission_confirmation(params)
   end
 
   def confirmation_instructions(record, token, opts = {})
-    contact = AccountService.get(record.salesforce_contact_id)
-    @name = if contact.present?
-              "#{contact['firstName']} #{contact['lastName']}"
-            else
-              record.email
-            end
-    super
+    load_salesforce_contact(record)
+    if record.pending_reconfirmation?
```

Comment:
nice re-use of emailer method. Is `pending_confirmation` a devise method that is determined by `email_changed?`.